dev-util/debugedit/debugedit-5.0-r3.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2023 Gentoo Authors
```

Comment:
No `git mv` for this please - new revision in ~arch.

Reply:
Done